pkg/console/subresource/deployment/deployment.go

```diff
@@ -74,6 +76,7 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 		infrastructureConfigResourceVersionAnnotation:        infrastructureConfig.GetResourceVersion(),
 		secretResourceVersionAnnotation:                      sec.GetResourceVersion(),
 		consoleImageAnnotation:                               util.GetImageEnv("CONSOLE_IMAGE"),
+		workloadManagementAnnotation:                         workloadManagementAnnotationValue,
```

Comment:
This will set the annotation on the `Deployment`. From the enhancement I was under impression that the `workload.openshift.io/management` should be set only on pods. Is that right ?

Reply:
You are correct that the machinery managing the workloads will only look at the pod. There's a comment on line 81 about tracking annotations on the deployment in case they change. The same annotation struct created on line 70 and used on line 83 is also used in the pod spec on line 107. Do we need the 2 sets of annotations to be the same in order to manage the update in the way that comment on line 81 implies?

Reply:
> Do we need the 2 sets of annotations to be the same in order to manage the update in the way that comment on line 81 implies?

I would rather have two sets of annotations. One `deploymentAnnotations` for deployment and second `podsAnnotations` for it's pods, where the pods annotation would include also the new annotation.

Reply:
done